function-client/src/main/java/io/micronaut/function/client/http/HttpFunctionExecutor.java

```diff
@@ -48,13 +48,13 @@
 @Singleton
 public class HttpFunctionExecutor<I, O> implements FunctionInvoker<I, O>, Closeable, FunctionInvokerChooser {
 
-    private final RxHttpClient httpClient;
+    private final ReactorHttpClient httpClient;
 
     /**
      * Constructor.
      * @param httpClient  The HTTP client
      */
-    public HttpFunctionExecutor(RxHttpClient httpClient) {
+    public HttpFunctionExecutor(ReactorHttpClient httpClient) {
```

Comment:
Can this be changed to just HttpClient ?

Reply:
Changed to use `HttpClient`